Description: ghc 7.4 class constraints
 This patch has been emailed to upstream Caylee Hogg.
Author: Clint Adams <clint@debian.org>

Index: haskell-crypto-4.2.5/Data/Digest/SHA2.hs
===================================================================
--- haskell-crypto-4.2.5.orig/Data/Digest/SHA2.hs	2012-10-19 23:38:06.092295145 +0200
+++ haskell-crypto-4.2.5/Data/Digest/SHA2.hs	2012-10-19 23:38:18.804294678 +0200
@@ -25,7 +25,7 @@
 ch x y z = (x .&. y) `xor` (complement x .&. z)
 maj x y z = (x .&. y) `xor` (x .&. z) `xor` (y .&. z)

-class (Bits w) => ShaData w where
+class (Bits w, Num w) => ShaData w where
   bigSigma0 :: w -> w
   bigSigma1 :: w -> w
   smallSigma0 :: w -> w
